perma_web/perma/views/common.py

```diff
@@ -206,40 +208,77 @@ def contact(request):
     """
     Our contact form page
     """
+    def affiliation_string():
+        affiliation_string = ''
+        if request.user.is_authenticated():
+            if request.user.registrar:
+                affiliation_string = "{} (Registrar)".format(request.user.registrar.name)
+            else:
+                affiliations = ["{} ({})".format(org.name, org.registrar.name) for org in request.user.organizations.all().order_by('registrar')]
+                if affiliations:
+                    affiliation_string = ', '.join(affiliations)
+        return affiliation_string
+
+    def handle_registrar_fields(form):
+        if getattr(request.user, 'is_organization_user', ''):
+            registrars = set(org.registrar for org in request.user.organizations.all())
+            if len(registrars) > 1:
+                form.fields['registrar'].choices = [(registrar.email, registrar.name) for registrar in registrars]
+            if len(registrars) == 1:
+                form.fields['registrar'].widget = widgets.HiddenInput()
+                email = registrars.pop().email
+                form.fields['registrar'].initial = email
+                form.fields['registrar'].choices = [(email,email)]
+        else:
+            del form.fields['registrar']
+        return form
+
+    def should_send_to_registrar():
```

Comment:
Good idea. In the design phase where we had a checkbox for "Is this a bug?" on the form, this function included "and not is_bug", so (at least some version of) it had to be here rather than in the model, but now.... no need! And thanks for the tip on AnonymousUser.